### DISCUSSION TITLE
How we can remove the unwanted space between the link and the shapes ?

### DISCUSSION THREAD

**Discussion by Lijesh-k** [clientIO/joint#2020] [Q&A]
![Screenshot 2023-02-03 162028](https://user-images.githubusercontent.com/122600888/216584321-489ab0d1-7728-4c6a-966a-3e383982b6f0.png)
![Screenshot 2023-02-03 162058](https://user-images.githubusercontent.com/122600888/216584345-f9ad9188-b18d-44e6-b75e-29aadfcd1644.png)

**Comment by kumilingus** [comment]
```js
paper.options.defaultConnectionPoint = { name: 'boundary' } 
```
- See `boundary` connection point [docs](https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#connectionPoints.boundary).

You can further specify the `magnet` by using the `magnetSelector`.
- See [discussion](https://github.com/clientIO/joint/discussions/1860).
- See `magnetSelector` [docs](https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.attributes.magnetSelector).
